packages/core/src/common/externals/BrowserExternals.ts

```diff
@@ -148,7 +148,7 @@ class BrowserFileSystem implements ExternalFileSystem {
 		delete this.states[location]
 		this.saveStates()
 	}
-	watch(_location: FsLocation): FsWatcher {
+	watch(_location: FsLocation[]): FsWatcher {
```

Comment:
nit (same for the implementation signatures): 
```suggestion
	watch(_locations: FsLocation[]): FsWatcher {
```